packages/lexical-editor/src/utils/join-classes.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export default function joinClasses(...args: Array<string | boolean | null | undefined>) {
+    return args.filter(Boolean).join(" ");
+}
```

Comment:
For operations on classes, we already use the library called `classnames` so you can use that directly, instead of creating these utility functions.

Reply:
This function was leftover from the original lexical editor code. This function is not used so I removed this file. 
Thanks for the info about `classnames`